types/src/transaction/mod.rs

```diff
@@ -1233,6 +1233,38 @@ impl TransactionOutputProvider for TransactionOutput {
     }
 }
 
+#[derive(Debug)]
+pub struct BlockOutput<Output: Debug> {
```

Comment:
Given this file is already too large, shall we put this struct into a separate file?